lib/src/asset/id.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```

Comment:
You can probably ignore this file, no changes currently from the barback version